### DISCUSSION TITLE
Update EEPROM for Compute Module 4 | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#803] [Q&A]
# Update EEPROM for Compute Module 4 | Seeed Studio Wiki

Update eeprom

https://wiki.seeedstudio.com/Raspberry_pi_CM4_update_eeprom/

**Comment by mosby333** [comment]
The CURL Commands need to be updated the github commands no longer work... The /directories don't exist anymore...

**Comment by FeliceGranito** [comment]
Could you please provide the right https link to the latest bin file  for the wget command ?
The actual one is wrong
wget -O pieeprom.original.bin https://github.com/raspberrypi/rpi-eeprom/blob/master/firmware/stable/pieeprom-2023-01-11.bin
